controllers/apply-next/simple/form-model.js

```diff
@@ -3,6 +3,15 @@ const path = require('path');
 const { get, includes, values } = require('lodash');
 const { check } = require('express-validator/check');
 const processor = require('./processor');
+const moment = require('moment');
+
+const minApplicantAge = 16;
+
+// Allows us to use postcode validation on the client-side
+// via https://github.com/chriso/validator.js/blob/master/lib/isPostalCode.js#L54
+// we have to double-escape the regex patterns here
+// to output it as a string for the HTML pattern attribute
+const POSTCODE_PATTERN = '(gir\\s?0aa|[a-zA-Z]{1,2}\\d[\\da-zA-Z]?\\s?(\\d[a-zA-Z]{2})?)';
```

Comment:
Took this from the Building Connections code – means on the client-side you have to enter a postcode format. Might be redundant once we do the API/lookup side, but useful for now.